lib/sensu/spawn.rb

```diff
@@ -159,8 +159,13 @@ def child_process(command, options={})
         end
         [output, child.exit_code]
       rescue ChildProcess::TimeoutError
-        child.stop rescue nil
-        ["Execution timed out", 2]
+        output = "Execution timed out"
+        begin
+          child.stop
+        rescue => error
```

Comment:
best to specify the exception class rather than using the implicit `StandardError`

Reply:
@majormoses I'd rather it be a catch all.

Reply:
@majormoses in the unlikely event Ruby changes the exception classes and/or hierarchy, not specifying a class here could be more beneficial. It's a 💩 shoot.